src/main/docker/env.bash

```diff
@@ -13,5 +13,5 @@ export REMREM_GENERATE_PORT=8095
 export REMREM_PUBLISH_PORT=8096
 
 export REMREM_PUBLISH_RABBITMQ_INSTANCES_LIST="[\
-{ \"mp\": \"eiffelsemantics\", \"host\": \"rabbitmq\", \"port\": \"5672\", \"username\": \"myuser\", \"password\": \"myuser\", \"tls\": \"\", \"exchangeName\": \"ei-exchange\", \"domainId\": \"ei-domain\", \"createExchangeIfNotExisting\":true }\
+{ \"mp\": \"eiffelsemantics\", \"host\": \"rabbitmq\", \"port\": \"5672\", \"username\": \"myuser\", \"password\": \"myuser\", \"tls\": \"\", \"exchangeName\": \"eiffel.poc\", \"domainId\": \"ei-domain\", \"createExchangeIfNotExisting\":true }\
```

Comment:
can we add the default configurations instead of anything specific, like localhost, guest , guest, etc

Reply:
I don't quite understand what you mean?

Also, i just changed one value here, i did not add the entire line.

Reply:
someone who tries to run the Integration tests should not see the tests failing, so can we use more specific naming instead of rabbitmq, ei-poc, etc?

Reply:
I think you mean more specific names instad of ei-poc ... We will change it in next commit.